peachjam/templates/peachjam/_ratifications.html

```diff
@@ -0,0 +1,36 @@
+<div>
```

Comment:
please ensure consistent indentation -- you also don't need this outer div.

Reply:
Changed.